test/FunctionalTests/Microsoft.OData.Core.Tests/JsonLight/ODataJsonLightResourceSerializerTests.cs

```diff
@@ -217,12 +217,12 @@ public async Task WriteResourceEndMetadataPropertiesAsync_WritesNavigationAndStr
                         new NullDuplicatePropertyNameChecker());
                 });
 
-            Assert.Equal("{\"BestSeller@odata.associationLink\":\"http://tempuri.org/Categories(1)/associationLink\"," +

```

Comment:
The changes here are mostly cosmetic but I wanted the assigned values to capture the essence of navigation and association links as documented in the specification
http://docs.oasis-open.org/odata/odata-json-format/v4.01/csprd01/odata-json-format-v4.01-csprd01.html#_Toc471396277